Resources/Prototypes/Entities/Structures/Machines/lathe.yml

```
@@ -1546,3 +1546,37 @@
     emagStaticRecipes:
       - PizzaLeftArm
       - PizzaRightArm
+
+- type: entity
+  parent: BaseLathe
+  id: MoneyPrinter
+  name: counterfeiting station
+  description: An illegal machine that counterfeits spesos that act exactly like real ones. Security will look down on this.
```

```suggestion
  description: An illegal machine that counterfeits spesos that act exactly like real ones. Security will definitely want to know about this.
```
makes it clearer you are 100% not supposed to have this